bundles/org.eclipse.swt/Eclipse SWT Drag and Drop/gtk/org/eclipse/swt/dnd/TextTransfer.java

```diff
@@ -145,13 +147,13 @@ protected int[] getTypeIds() {
 	if(GTK.GTK4) {
 		return new int[] {(int) OS.G_TYPE_STRING()};
 	}
-	return new int[] {UTF8_STRING_ID, STRING_ID};
+	return new int[] {UTF8_STRING_ID, STRING_ID, TEXTPLAINUTF8_ID};
 }
 
 @Override
 protected String[] getTypeNames() {
 	if (OS.isX11()) {
-		return new String[] {UTF8_STRING, COMPOUND_TEXT, STRING};
+		return new String[] {UTF8_STRING, COMPOUND_TEXT, STRING, TEXTPLAINUTF8};
 	}
 	if(GTK.GTK4) {
 		return new String[] {"text/plain", STRING};
```

Comment:
Shouldn't it be added here also? Basically everywhere?

Reply:
it gives no meaning to add TEXTPLAINUTF8 in if (OS.isX11()) { (x11) section as the clipboard workes fine there without.
For if(GTK.GTK4) {  (GTK4) section I do not know, AFAIK there is no currently working builds of swt or eclipse gtk4 to test on ?